### ISSUE TITLE
Словарь синонимов названий городов

### ISSUE THREAD

**Issue by audetv** [terratensor/common-library-app#26]
Появилась мысль, оставляю для ПФУ, составить словарь наподобие концептуального словаря с синонимами названий городов, мест, населённых пунктов. Исторические названия городов, разговорные названия и пр.

**Comment by iprst** [comment]
Так [вот тут](https://github.com/terratensor/geomatrix/blob/main/datasets/all-cities-with-a-population.csv) уже часть этой работы сделана: у всех наименований городов есть официальные названия на большинстве языков мира. Не совсем то же самое, но логика в том же векторе, мы подобное обсуждали ранее. 

Добавлять локальные топонимы это мысль верная.

**Comment by iprst** [comment]
Плюс я так обдумал, кажется что в указанном вами векторе также лежит наша географическая библиотека, которую мы подготовили и библиотека вообще, в смысле поиск по любой библиотеке по сущностям «топоним». Видимо это всё в одном векторе. 

А почему идея о словаре в голову пришла?

**Comment by audetv** [comment]
Да, думаю это лежит в одном векторе, сущность топоним, согласен. Идея пришла в голову, так как в обсуждении был Сергиев Посад. Ранее он назывался Загорск. В этом городе я бывал в детстве, проводил каникулы, поэтому внимание привлёк сразу. И в нашей библиотеке не найти упомянутый завод по новому именованию города. А по метрике я увидел, что кто-то пытался найти именно как завод в Сергиевом Посаде. Решил зафиксировать идею, но она действительно глубже чем просто словарь для военно-исторической библиотеки. Теперь эта идея тоже работает, думается в фоне)

**Comment by iprst** [comment]
Есть топонимические справочники, очень подробные, локальные, с примерами названий на местных диалектах и тд. Они есть по многим странам и регионам, в большой библиотеке они попадаются, но полноты на таком уровне добиться сложно. Полу-полнота впрочем тоже хорошо.
